synfig-studio/src/gui/CMakeLists.txt

```diff
@@ -28,7 +28,12 @@ endif()
 ## Targets
 ##
 
-add_executable(synfigstudio main.cpp)
+add_executable(synfigstudio main.cpp ${CMAKE_CURRENT_BINARY_DIR}/../../ui/resource.c)
+add_dependencies(synfigstudio build_resource)
+set_source_files_properties(
+	${CMAKE_CURRENT_BINARY_DIR}/../../ui/resource.c
```

Comment:
Maybe better rename it to `custom_widgets.c`?

Reply:
Name sounds nice but in project structure sense it's actually a resource file even though it's C

Reply:
But since it's generated outside synfig source, I think it's OK to name it that way. Do I proceed with the change?

Reply:
`custom_widgets_resource` maybe?

Reply:
Ok, I will look further, maybe I misunderstood the purpose of this file

Reply:
> custom_widgets_resource maybe?

Sounds better. Committing the changes...